xmtp_mls/src/storage/errors.rs

```diff
@@ -14,6 +14,8 @@ pub enum StorageError {
     Store(String),
     #[error("serialization error")]
     SerializationError,
+    #[error("deserialization error")]
+    DeserializationError,
```

Comment:
This is more something that doesn't _really_ matter, but imo I prefer errors like 
`Error::Deserialization`

Rather then including the redundant 'Error' on the end, since the enum will  be used in-code, and saves some line-space.